packages/block-editor/src/components/block-mobile-toolbar/index.native.js

```diff
@@ -19,9 +19,18 @@ import styles from './style.scss';
 import BlockMover from '../block-mover';
 import { BlockSettingsButton } from '../block-settings';
 
-const BlockMobileToolbar = ( { clientId, onDelete, order } ) => (
+const BlockMobileToolbar = ( {
+	clientId,
+	onDelete,
+	order,
+	customOnDelete,
```

Comment:
WDYT about removing the `customOnDelete` and use just `onDelete` instead? I know the `onDelete` is declared already in compose here https://github.com/WordPress/gutenberg/pull/19013/files#diff-b20ce6a53833ca6ec70166ea994cb910R63 but we probably can get `onDelete` from `ownProps` and in return do something like:
```js
onDelete: ownProps.onDelete || () => {
  Keyboard.dismiss();
  removeBlock( clientId, rootClientId );
}
```

Reply:
that make sense to me.